zend_abstract_interface/config/config_ini.h

```diff
@@ -0,0 +1,33 @@
+#ifndef ZAI_CONFIG_INI_H
+#define ZAI_CONFIG_INI_H
+
+#include <main/php.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#include "../env/env.h"
+#include "../zai_string/string.h"
+
+#include "config.h"
+
+#endif  // ZAI_CONFIG_INI_H
```

Comment:
Is this not at the bottom for a specific reason?

Reply:
It is, because of the recursive include ... config.h wants to include config_ini.h. But config_ini.h references types only defined in config.h. If you put this at the bottom compilation will fail :-)